packages/runtime/src/useCases/anonymous/CheckBackboneCompatibilityUseCase.ts

```diff
@@ -0,0 +1,15 @@
+import { Result } from "@js-soft/ts-utils";
+import { BackboneCompatibilityController } from "@nmshd/transport";
+import { Inject } from "typescript-ioc";
+import { UseCase } from "../common";
+
+export class CheckBackboneCompatibilityUseCase extends UseCase<void, void> {
```

Comment:
void as successful result and throwing an error if it's not successful is very hard to process. Why not return a proper response that has 

1. a boolean to know if it's successful
2. an explanation on why it's not successful if it's not.